ui/temboardui/templates/base.html

```diff
@@ -21,54 +21,55 @@
     <script type="module" src="{{ vitejs.url_for('temboard.js') }}"></script>
     <!-- Navigation -->
     <div class="navbar bg-primary navbar-static-top navbar-expand sticky-top p-0" data-bs-theme="dark">
-      <div class="collapse navbar-collapse">
-        {% if 'instance' in globals() %}
-        <a class="navbar-brand ps-0 ms-3 me-0" href id="menu-collapse">
-          <i class="fa fa-bars"></i>
-        </a>
+      {% if 'instance' in globals() %}
+      <a class="navbar-text fs-6 ms-3 text-white" href id="menu-collapse">
+        <i class="fa fa-bars"></i>
+      </a>
+      {% end %}
+      <a class="navbar-brand fs-5 p-0 ms-3" href="/home">temBoard</a>
+      {% if 'instance' in globals() %}
+      {% set d = instance.discover or {} %}
+      <span class="navbar-text p-0 fs-5 text-white text-truncate">
```

Comment:
I tried to understand why the brand "temBoard" text was shrinked compared to master version. Then I played with the `fs-x` classes added here, and surprisingly, removing them doesn't have any effect.

Even more surprisingly, the dev tools tell me that the font-size is supposed to be 17.5px even though it is effectively small.

![Screenshot from 2024-08-23 10-18-20](https://github.com/user-attachments/assets/f7292148-0cc4-4064-9f80-744f62be8c85)

I think the `navbar-collapse` removal has something to do with this.